nginx.master.conf

```diff
@@ -302,9 +302,14 @@ http {
             rewrite ^/agent/[0-9a-zA-Z-]+(.*)$ $1 break;
 
             more_clear_input_headers Accept-Encoding;
-            include common/server-sent-events.conf;
             include common/proxy-headers.conf;
 
+            # Supports chunked encoding and stream for debugging API
+            proxy_http_version 1.1;
+            proxy_request_buffering off;
+            proxy_buffering off;
+            chunked_transfer_encoding on;
+
```

Comment:
Maybe consider putting this into a file called `common/streamable.conf` for reusability. But otherwise, LGTM